namespaces/live.cloud-platform.service.justice.gov.uk/hale-platform-prod/resources/justiceinspectorates.tf

```diff
@@ -21,3 +21,83 @@ resource "kubernetes_secret" "justiceinspectorates_route53_zone" {
   }
 }
+
+resource "aws_route53_record" "justiceinspectorates_route53_cname_record_main_www" {
+  zone_id = aws_route53_zone.justiceinspectorates_route53_zone.zone_id
+  name    = "www.justiceinspectorates.gov.uk"
+  type    = "CNAME"
+  ttl     = "3600"
+  records = ["inspectorates.live.bangdynamics.com"]
+}
+
+resource "aws_route53_record" "justiceinspectorates_route53_cname_record_main" {
+  zone_id = aws_route53_zone.justiceinspectorates_route53_zone.zone_id
+  name    = "justiceinspectorates.gov.uk"
+  type    = "CNAME"
+  ttl     = "3600"
+  records = ["inspectorates.live.bangdynamics.com"]
+}
+
+resource "aws_route53_record" "justiceinspectorates_route53_cname_record_assets" {
+  zone_id = aws_route53_zone.justiceinspectorates_route53_zone.zone_id
+  name    = "assets-hmicfrs.justiceinspectorates.gov.uk"
+  type    = "CNAME"
+  ttl     = "3600"
+  records = ["dzm0ui1toev7.cloudfront.net"]
+}
+
+resource "aws_route53_record" "justiceinspectorates_route53_cname_record_hmicfrs" {
+  zone_id = aws_route53_zone.justiceinspectorates_route53_zone.zone_id
+  name    = "hmicfrs.justiceinspectorates.gov.uk"
+  type    = "CNAME"
+  ttl     = "3600"
+  records = ["web-1202467189.eu-west-2.elb.amazonaws.com"]
+}
+
+resource "aws_route53_record" "justiceinspectorates_route53_cname_record_hmiprobationeforms" {
+  zone_id = aws_route53_zone.justiceinspectorates_route53_zone.zone_id
+  name    = "hmiprobationeforms.justiceinspectorates.gov.uk"
+  type    = "CNAME"
+  ttl     = "3600"
+  records = ["hmiprobationeforms.live.bangdynamics.com"]
+}
+
+resource "aws_route53_record" "justiceinspectorates_route53_cname_record_monitoringportal" {
+  zone_id = aws_route53_zone.justiceinspectorates_route53_zone.zone_id
+  name    = "monitoringportal.justiceinspectorates.gov.uk"
+  type    = "CNAME"
+  ttl     = "3600"
+  records = ["hmicfrsrecommendations.live.bangdynamics.com"]
+}
+
+resource "aws_route53_record" "justiceinspectorates_route53_cname_record_hmicfrs_acm" {
+  zone_id = aws_route53_zone.justiceinspectorates_route53_zone.zone_id
+  name    = "_4db237fe0679b5055c854f480873647d.hmicfrs.justiceinspectorates.gov.uk"
+  type    = "CNAME"
+  ttl     = "3600"
+  records = ["_2326da8b5885263e394ef0fdfa48c4b5.hnyzmxtzsz.acm-validations.aws"]
+}
+
+resource "aws_route53_record" "justiceinspectorates_route53_cname_record_assets_acm" {
+  zone_id = aws_route53_zone.justiceinspectorates_route53_zone.zone_id
+  name    = "_1bb6da7fc8ff3c4473d3f446d943f10a.assets-hmicfrs.justiceinspectorates.gov.uk"
+  type    = "CNAME"
+  ttl     = "3600"
+  records = ["_13864bc89cfac506d876bcba6914a2ba.kmjqhnbgnp.acm-validations.aws"]
+}
+
+resource "aws_route53_record" "justiceinspectorates_route53_mx_record_main" {
+  zone_id = aws_route53_zone.justiceinspectorates_route53_zone.zone_id
+  name    = "justiceinspectorates.gov.uk"
+  type    = "MX"
+  ttl     = "3600"
+  records = ["mx1.bangdynamics.com", "mx2.bangdynamics.com"]
```

Comment:
Please update with priority values should be: 
`records = ["10 mx1.bangdynamics.com", "20 mx2.bangdynamics.com"]`